asteroidsyncserviced/bluez/bluezclient.h

```diff
@@ -57,11 +56,7 @@ private slots:
 
 private:
     bool isAsteroidOSWatch(const QStringList uuids) const;
-    QDBusConnection m_dbus;
-    DBusObjectManagerInterface m_bluezManager;
-    BluezAdapter1 *m_bluezAdapter = nullptr;
-    FreeDesktopProperties *m_bluezAdapterProperties = nullptr;
-
+    BluezClientImpl *bt = nullptr;
```

Comment:
If you make this a std::unique_ptr it will be deleted automatically and you don't need a BluezClient destructor